Dfe.SchoolAccount.Web/Views/Home/Index.cshtml

```diff
@@ -1,8 +1,9 @@
-﻿@{
+﻿@inject IViewLocalizer viewLocalizer
```

Comment:
Apologies for the pedantry, but in C# should we favour Pascal casing for variables? So in this case, it would be `ViewLocalizer`